ansible/roles/test/tasks/pfc_wd/pfc_wd_config.j2

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Consider create a pfcwd folder under template and move all j2 there?

Reply:
Done